src/java/za/co/mip/ablduck/javadoc/JavadocListener.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package za.co.mip.ablduck;
+
+import java.util.ArrayList;
+
+public class JavadocListener extends JavadocParserBaseListener {
+    ArrayList tags = new ArrayList();
+    
+    public ArrayList getTags() {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1319)